pom.xml

```diff
@@ -181,6 +186,55 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<version>1.7</version>
+				<executions>
+					<execution>
+						<id>jarbundler-gui</id>
+						<phase>package</phase>
+						<goals>
+							<goal>run</goal>
+						</goals>
+						<configuration>
+							<target>
+								<taskdef name="jarbundler" classname="com.ultramixer.jarbundler.JarBundler" />
```

Comment:
I'm not all entirely familiar in the world of OSX, however how much does https://github.com/federkasten/appbundle-maven-plugin differ from this plugin and would it be better to embed the jvm, such as launch4j.